actix-http/src/h1/encoder.rs

```diff
@@ -273,6 +263,25 @@ pub(crate) trait MessageType: Sized {
 
         Ok(())
     }
+
+    #[inline]
```

Comment:
inlining happens inside crate boundaries by default (at compilers discretion), so either remove or `#[inline(always)]` if you think it's appropirate